src/work-allocation/containers/available-tasks/available-tasks.component.ts

```diff
@@ -1,6 +1,8 @@
 import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
 
 import { ConfigConstants, ListConstants, SortConstants } from '../../components/constants';
+import { ConfigConstants as ConfigConstantsRelease2 } from 'src/work-allocation/release2/components/constants';
```

Comment:
Should not import from `src/`. And that `Release2` suffix is horrible! 😱

Reply:
If you have a suffix you think is more appropriate let me know. I have asked a couple of times in meetings and noone had an issue with it. It can always be changed down the road once the release 2 feature branch is more established

Reply:
One issue with suffixes going on the _new_ stuff is that when the old stuff gets killed off - and it will - you're left with all the suffixes that no longer make sense. If you absolutely _have_ to have suffixes, I think it should go on the old stuff so the suffixes die when they do.

In this particular instance, I don't even see why there's a need for a copy of `ConfigConstants`. It doesn't appear to be being used in all instances where there's a feature toggle switch either. 🤷